src/types/any.rs

```diff
@@ -169,16 +216,40 @@ impl PyAny {
         })
     }
 
-    /// Compares two Python objects.
+    /// Tests whether two Python objects obey a given [`CompareOp`].
     ///
     /// Depending on the value of `compare_op`, this is equivalent to one of the
     /// following Python expressions:
-    ///   * CompareOp::Eq: `self == other`
-    ///   * CompareOp::Ne: `self != other`
-    ///   * CompareOp::Lt: `self < other`
-    ///   * CompareOp::Le: `self <= other`
-    ///   * CompareOp::Gt: `self > other`
-    ///   * CompareOp::Ge: `self >= other`
+    ///
+    /// <div style="width:1px">
+    ///
+    /// | `compare_op` | <span style="white-space: pre">Python expression</span> |
```

Comment:
Why is that `span` needed?

Reply:
Combined with the div that's 1px, it shrinks the table as much as possible. Otherwise it expands the table as big as it can get, which I think looks odd.